front/src/common/Map/Search/MapSearchLine.tsx

```diff
@@ -40,11 +40,10 @@ const MapSearchLine: React.FC<MapSearchLineProps> = ({
     id: 0,
     lineCode: 0,
   });
-  const [skip, setSkip] = useState<boolean>(true);
 
   const { data: trackZones } = osrdEditoastApi.useGetInfraByIdLinesAndLineCodeBboxQuery(
     dataTrackZone,
-    { skip }
+    { skip: !infraID }
```

Comment:
Nice, thanks ! :)

Reply:
Ah can you initialize trackZones to undefined and skip if trackZones is undefined too ? Otherwise, there is a useless api call when the component is mounted
![image](https://github.com/osrd-project/osrd/assets/45000526/71466228-20cb-4d68-ba47-d18ad9e6ba2c)

```suggestion
    { skip: !infraID || !dataTrackZone }
```